scripts/bin/unpack_dvd

```diff
@@ -339,6 +362,46 @@ def unpack_pxeboot_from_iso(discattrs, versiondir, isoinfo, options)
   puts "Unpacking PXE boot image files complete."
 end
 
+def fail_on_unspecific_versiondir(versiondir, options, discattrs)
+  if versiondir.split('.').size <= 1
+    if options[:version] == versiondir
+       warn <<~MSG
+         WARNING: unspecific versioned directory name ('#{versiondir}'),
+                 (Allowing because it was passed in by option)
+
+       MSG
+       return
+    end
+
+    puts <<~MSG
+  ERROR:
+
+  The versioned directory name ('#{versiondir}') this script is configured to unpack into
+  is not specific enough to be safe; if used, this ISO\'s data may overwrite/be
+  overwritten by other unpacked ISOs.
+
+  Specific release versions must contain at least one dot (.) character.
+
+  The version '#{versiondir}' may have been automatically determined from the ISO\'s
+  metadata; some OS distributions (CentOS) only provide the major OS release
+  version in their ISOs\' metadata.
+
+  WORKAROUND:
+
+  Run this script again with `-v` and specify a more specific version.  For
+  instance, when unpacking  ISO CentOS-7-x86_64-DVD-2003.iso, you could run:
+
+     unpack_dvd -v 7.0.2003 CentOS-7-x86_64-DVD-2003.iso
```

Comment:
When this is used to unpack a SIMP ISO (i.e., follow our upgrade instructions), it is less clear what version to use.  Unless I am missing something, I would have to go back to the documentation to figure out which version of CentOS was used to create the SIMP ISO.

Reply:
Also, why is it that the following happens when I unpack a SIMP ISO during the upgrade:
* /var/www/yum/CentOS is updated to contain a new x.y.z directory (e.g., '7.8.2003'), but the major version link does not change.
* /var/simp/environments/production/rsync/RedHat/Global/tftpboot/linux-install/<major version> gets updated, instead of creating centos-x.y.z and moving the link.

I do not understand why part of the program knows about the x.y.z version and the rest of it does not.

Reply:
This patch didn't change any of those behaviors; it just makes `-v` mandatory if the detected version number isn't specific enough.

Reply:
I guess what I am saying is that the warning just adds to the layers of confusion on how the program works.